trojsten/submit/views.py

```diff
@@ -56,8 +55,10 @@ def protocol_data(submit, force_show_details=False):
             "compileLogPresent": protocol.compile_log is not None,
             "compileLog": protocol.compile_log,
         }
+
         tests = [
             {
+                "set": runtest.name.split(".")[0],
```

Comment:
Toto je tu pouzite 2x mozno sa to da extrahovat do funkcie s prihodnym nazvom. 

Chceme osetrit aj case ak je name prazdny?

Reply:
Extrahujem, ak je name prazdny, set bude `""`:
![image](https://user-images.githubusercontent.com/11409143/66598250-e9b8ae00-eba0-11e9-821a-a96017a0b914.png)